controlplane/kubeadm/api/v1alpha4/kubeadm_control_plane_types.go

```diff
@@ -28,6 +28,10 @@ import (
 const (
 	KubeadmControlPlaneFinalizer = "kubeadm.controlplane.cluster.x-k8s.io"
 
+	// DeleteControlPlaneMachineAnnotation annotation deletes Control Plane machines with delete
+	// machine annotation key during the scaling down of KCP
+	DeleteControlPlaneMachineAnnotation = "kubeadm.controlplane.cluster.x-k8s.io/delete-machine"
```

Comment:
Could we just reuse the `DeleteMachineAnnotation` defined in the core types?

Reply:
I just realized that somehow that constant is defined in the `controllers` package. We should move it under the `api/v1alpha4` package instead, and then we should be able to reuse it here

Reply:
+1

Reply:
Thanks for the suggestion, I have made the changes accordingly.